src/main/java/io/appium/java_client/DefaultGenericMobileDriver.java

```diff
@@ -59,6 +59,14 @@ public DefaultGenericMobileDriver(CommandExecutor executor, Capabilities desired
         return (T) super.findElement(by);
     }
 
+    @Override public T findElement(String by, String using) {
+        return (T) super.findElement(by, using);
+    }
+
+    @Override public List findElements(String by, String using) {
```

Comment:
This method shpuld be overriden at AppiumDriver as well. Just take a look at
https://github.com/appium/java-client/blob/master/src/main/java/io/appium/java_client/AppiumDriver.java#L223
for instance.